app/controllers/membership_applications_controller.rb

```diff
@@ -126,20 +138,32 @@ def authorize_membership_application
   end
 
 
-  def new_upload_file(upload_file_param)
-    if upload_file_param['actual_files']
-      upload_file_param['actual_files'].each do |upload_file|
+  def new_file_uploaded(params)
+
+    successful = true
+
+    if params.fetch('uploaded_file', false) && params['uploaded_file'].fetch('actual_files', false)
+
+      params['uploaded_file']['actual_files'].each do |uploaded_file|
```

Comment:
There is a lot going on in these three lines and it's a bit tough to figure out what's going on with a quick read.

Perhaps something like this?
```
if (uploaded_files = params['uploaded_file'])

  uploaded_files['actual_files'].each do |uploaded_file|
```